HISTORY.md

```diff
@@ -15,6 +15,7 @@
 ### New Features
 * DB identity (`db_id`) and DB session identity (`db_session_id`) are added to table properties and stored in SST files. SST files generated from SstFileWriter and Repairer have DB identity “SST Writer” and “DB Repairer”, respectively. Their DB session IDs are generated in the same way as `DB::GetDbSessionId`. The session ID for SstFileWriter (resp., Repairer) resets every time `SstFileWriter::Open` (resp., `Repairer::Run`) is called.
 * Added experimental option BlockBasedTableOptions::optimize_filters_for_memory for reducing allocated memory size of Bloom filters (~10% savings with Jemalloc) while preserving the same general accuracy. To have an effect, the option requires format_version=5 and malloc_usable_size. Enabling this option is forward and backward compatible with existing format_version=5.
+* `BackupableDBOptions::new_naming_for_backup_files` is added. This option is true by default. When it is true, backup table filenames are of the form `<file_number>_<crc32c>_<db_session_id>.sst` as opposed to `<file_number>_<crc32c>_<file_size>.sst`. When there is no `db_session_id` available in the table file, we use `file_size` as a fallback. Note that when this option is true, it comes into effect only when both `share_files_with_checksum` and `share_table_files` are true.
```

Comment:
I am also in favor of a name indicating it is only for legacy compatibility. Another benefit of that naming style is it'll be less surprising if we deprecate it in the future. Ideally we will find nobody's relying on the legacy naming convention and then remove the option in the next major release (7.0).

Maybe `legacy_sst_naming_no_session_id` or something.

Reply:
Thanks for the suggestion! Actually, @pdillinger has a similar suggestion with consideration of potential extension of the naming scheme in the future. I will probably create a follow-up PR for it soon (`enum option share_files_with_checksum_naming = {kChecksumAndFileSize, kChecksumAndDbSessionId}`). So we may leave it as it is for the moment :)

Reply:
(I promise I'll create a follow-up soon :P)